source/designers-guide/eslint-for-plugins/index.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: ESLint for plugins
+github_link: designers-guide/find-smarty-blocks/index.md
+indexed: true
+group: Frontend Guides
+subgroup: General Resources
+menu_title: ESLint for plugins
+---
+
+### Introduction
+Shopware features an automatic static code analyses tool called [ESLint](http://eslint.org/). Combined with a shared configuration called [standardjs](https://standardjs.com/) we provided a convenient way to ensure a consistent code style in the core & in our plugins.
+
+In this guide we'll talking a look on how you as a Shopware plugin developer can use the benefits of ESLint in your own plugin.
+
+### Installation
+ESLint has Node.js and NPM as an dependency you have to install beforehand. If you haven't installed them, please go ahead and install it using the provided [download methods](https://nodejs.org/en/download/current/).
```

Comment:
ESLint has Node.js and NPM as dependencies, which you have to install beforehand.